spec/rails_helper.rb

```diff
@@ -8,6 +8,7 @@
 
 require 'spec_helper'
 require File.expand_path('../config/environment', __dir__)
+Rails.application.eager_load!
```

Comment:
This seems like it could slow down tests. Although, I don't think we have that many tests so maybe the pros beat the cons here.